Tech@HubAPI/Tech@HubAPI/Controllers/RepositoryController.cs

```diff
@@ -42,6 +43,24 @@ public ActionResult<List<Branch>> GetBranches([FromBody] GetBranchesForm form)
             return branches;
         }
 
+        [HttpPost]
+        [Route("getdirectorylisting")]
```

Comment:
We need to change this to be a Get Request utilizing query parameters.

Reply:
https://code-maze.com/aspnetcore-required-query-string-parameters/

Reply:
@Heckin-Doggo #77